applications/asset_tracker/Kconfig

```diff
@@ -8,6 +8,14 @@ menu "Asset tracker"
 
 rsource "src/ui/Kconfig"
 
+config APPLICATION_WORKQUEUE_STACK_SIZE
+	int "Application workqueue stack size"
+	default 2048
+
+config APPLICATION_WORKQUEUE_PRIORITY
+	int "Application workqueue priority"
+	default -1
```

Comment:
I assume the intention is to default to the same as the system workqueue, so could use `default SYSTEM_WORKQUEUE_PRIORITY`